packages/plugin-bold/BoldPlugin.ts

```diff
@@ -18,4 +22,15 @@ export class BoldPlugin extends JWPlugin {
         newContext.format = newFormat;
         return [newContext, new Map()];
     }
+    /**
+     * Apply the `format` to the range.
+     *
+     * If the range is collapsed, set the format on the range so we know in the next insert
+     * which format should be used.
+     */
+    applyFormat(params: FormatParams): void {
+        if (params.formatName === 'bold') {
+            this.editor.vDocument.applyFormat(new Bold());
```

Comment:
Ah ah, see, you actually do create the `new` here, so the override of `applyFormat` isn't really useful since it only brings the conversion from `string` to `Format` when you could have given a `Format` since the start :p